framework/src/executioners/Executioner.C

```diff
@@ -51,7 +51,8 @@ Executioner::Executioner(const InputParameters & parameters)
         "_fe_problem_base", "This might happen if you don't have a mesh")),
     _feproblem_solve(this),
     _picard_solve(this),
-    _restart_file_base(getParam<FileNameNoExtension>("restart_file_base"))
+    _restart_file_base(getParam<FileNameNoExtension>("restart_file_base")),
+    _verbose(getParam<bool>("verbose"))
```

Comment:
Your description said that you moved the verbose flag from `Transient` to `FEProblemSolve`. It appears that you also moved it to `Executioner`. Is that necessary? Also, if it is, does it need to have an accessor that returns a writable reference? I can see that it did for `Transient` but it would be nice to know if that causes an issue or who is using that writable reference.

Reply:
`Executioner` gets its parameters from `FEProblemSolve` and `PicardSolve`, so it implicitly gets the `verbose` parameter. I put it in `FEProblemSolve` rather than directly in `Executioner` because it seemed to be the existing design. As for the writable reference, I had just assumed that `Transient` had needed it to be writable, but I don't see any instances of that being necessary, so I'll change that to non-writable.